### ISSUE TITLE
Tutorial/API Docs issues

### ISSUE THREAD

**Issue by brettz9** [projectfluent/fluent.js#120]
From my review of the tutorial and API docs:

1. It looks like http://projectfluent.org/fluent.js/fluent/ may need to be updated as its referring to http://projectfluent.io/ which is giving a 404
2. I see on [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat) one argument missing from http://projectfluent.org/fluent/guide/functions.html#datetime : `hourCycle`
3. Under the tutorial docs for Number, I am not clear on the meaning of this one sentence: "The second example may be used to pass additional formatting options to the NUMBER formatter for the purpose of choosing the correct plural category".

I think the above-referenced API documentation could also probably benefit from documenting the available built-in classes (`MessageArgument`, `MessageNumberArgument`, and `MessageDateTimeArgument`) for use as part of partial arguments within `format` (if not the other exports, `mapContextSync` and `CachedIterable`) since looking through the API docs alone didn't make this apparent.

**Comment by brettz9** [comment]
I've added a few more items to the list.

**Comment by brettz9** [comment]
While I understand it may take a little time to investigate these issues, I'd be most appreciative for a clarification now about the differences between the seemingly overlapping projects (and overlapping APIs), fluent-web (which isn't referenced on the main README), fluent-dom, and the separate l20n.js project (which incidentally sees like it should be dependent on fluent-dom but isn't). Is fluent-web meant to take over the role of l20n.js? I'd rather not be digging into the code of a project you're deprecating...

**Comment by stasm** [comment]
@brettz9 Thank you so much for the thorough review of the documentation! There sure is a lot of room for improvement :) I'll be working this week and next week on addressing the issues you've found.

To answer the questions from your second comment: yes, `fluent-web` is meant as the replacement for the now-deprecated `l20n.js`. I opened https://github.com/l20n/spec/issues/17 to properly mark `l20n.js` as deprecated and I'll update Fluent's READMEs (including `fluent-web`) accordingly.

**Comment by brettz9** [comment]
No worries--these were mostly small recommendations, and I am quite eager to see the web community (including myself) benefit from what appears to me to be such a very well thought out, experience-driven and comprehensive framework....

**Comment by brettz9** [comment]
Any chance of reviewing these?

**Comment by zbraniecki** [comment]
Assigning stas since he started reviewing it already. :)

**Comment by anlexN** [comment]
who can tell me fluentjs tutorial? i don't understanding how to use fluent in my web project!
seemingly, fluentjs is javascript specification, like ecmascript, not a framework.

**Comment by brettz9** [comment]
@anlexN : No, there is a framework for it though it defines a file format . For the framework, see especially https://github.com/projectfluent/fluent.js/tree/master/fluent . For the format of the file, see https://projectfluent.org/fluent/guide/ and https://projectfluent.org/play/

@stasm : Any chance of a review? Also, FYI, I've added a new item at no. 11 suggesting Terms mention their possible use for Unicode (like how XML entities can be used).
